README.md

```diff
@@ -10,3 +10,80 @@
 - Corentin
 
 [Docs Route API](/docs/api.md)
+
+## Installation
+
+Clone the project with the following command
+
+```
+git clone git@github.com:WildCodeSchool/reims-0918-js-tackle-swap.git
+```
+
+### BACK
+
+---
+
+#### Modules:
+
+First go to the back file and use the following command to install packages.
```

Comment:
what is a back file ? (I don't want to know actually, just give the directory name)